ibm/service/appconfiguration/data_source_ibm_app_config_collection.go

```diff
@@ -0,0 +1,234 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+package appconfiguration
+
+import (
+	"fmt"
+	"github.com/IBM/appconfiguration-go-admin-sdk/appconfigurationv1"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"log"
+)
+
+func DataSourceIBMAppConfigCollection() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceIbmAppConfigCollectionRead,
```

Comment:
Can we move to new feature of v2 SDK to use Context
https://www.terraform.io/plugin/sdkv2/guides/v2-upgrade-guide#more-support-for-context-context

Reply:
Sure, We will create separate issue for moving to v2 SDK for our service.